templates/partials/messages_list.html

```diff
@@ -4,7 +4,7 @@
       <a href="#messages/{{message.key}}">
         <div class="content">
           <div class="heading">
-            <h4>{{message.contact || message.from}}</h4>
+            <h4>{{message.contact || message.from || "Unknown sender" | translate}}</h4>
```

Comment:
TODO: you haven't added the translation and it should be defined as lowercase and dot separated, but I'm sure you were going to do that before merging anyway...